content/scripts/build.js

```diff
@@ -42,6 +43,10 @@ const ALL_TITLES_JSON_FILEPATH = path.join(
   "_all-titles.json"
 );
 
+function msLong(milliseconds) {
+  // https://github.com/vercel/ms/blob/master/readme.md
```

Comment:
nit: personally don't feel like links to docs need to live in code, but then again I do have an editor that does stuff like this:
![Screenshot (14)](https://user-images.githubusercontent.com/4051932/84407336-ff3c1700-ac0a-11ea-9b66-6747c3b41f50.png)

So maybe I'm biased that way